src/plugins/orangehrmLeavePlugin/Service/LeaveTypeService.php

```diff
@@ -80,6 +82,10 @@ public function getLeaveTypeAsArray(int $leaveTypeId): ?array
         if (!$leaveType instanceof LeaveType) {
             return null;
         }
-        return $this->getNormalizerService()->normalize(LeaveTypeModel::class, $leaveType);
+        $leaveType = $this->getNormalizerService()->normalize(LeaveTypeModel::class, $leaveType);
```

Comment:
@RajithaKumara I had to change this normalise function to concat a deleted tag to the leave type label since our select component don't support suffixes. might not be the best way :face_with_head_bandage:

Reply:
Okay. if there is no way to do in FE